elasticdl/python/collective_ops/communicator.py

```diff
@@ -0,0 +1,16 @@
+# TODO: This is dummy for now until the real implementation
+# has been open sourced
+
+
+class CollectiveCommunicator(object):
+    def __init__(self):
+        pass
+
+    def allreduce_average(self, data):
```

Comment:
Better to use `allreduce(self, data, op)` where `op` can be `sum`, `average`, etc.